docs/docsource/operation/XDM_GUI/file_operations/upload_files_to_buckets.rst

```diff
@@ -0,0 +1,51 @@
+Uploading Files to Buckets
+==========================
+
+**Prerequisites:** Before uploading data to a bucket, you must have a storage
+account associated with a user, and you must have created at least one bucket.
+
+#. Click the **Data Browser** tab to open the bucket view.
+
+   .. image:: ../../Graphics/xdm_ui_data_browser.png
+
+#. Select an account from the account pull-down menu.
+
+   .. image:: ../../Graphics/xdm_ui_account_selected.png
+
+#. Click the link of the bucket you will upload data to. 
+
+   .. image:: ../../Graphics/xdm_ui_bucket_select.png
+      :width: 75% 
+
+#. The bucket's contents (if any) are displayed.
+
+   .. image:: ../../Graphics/xdm_ui_bucket_contents.png
+      :width: 50%	      
```

Comment:
This image is rather small compared to the others. Was it done on purpose?

Reply:
Good spot. Fixed.